openedx/core/djangoapps/content_tagging/rest_api/v1/tests/test_views.py

```diff
@@ -219,12 +228,59 @@ def test_list_taxonomy(self, user_attr, enabled_parameter, org_name, expected_ta
         assert response.status_code == status.HTTP_200_OK
         self.assertEqual(set(t["name"] for t in response.data["results"]), set(expected_taxonomies))
 
+    def test_list_taxonomy_staff(self):
+        """
+        Tests that staff users see all taxonomies
+        """
+        # Default page_size=10, and so "tC1" and "tC2" appear on the second page
+        expected_taxonomies = ("ot1", "ot2", "st1", "st2", "t1", "t2", "tA1", "tA2", "tB1", "tB2")
+        self._test_list_taxonomy("staff", None, None, expected_taxonomies)
+
+    @ddt.data(
+        "content_creatorA",
+        "instructorA",
+        "libraryA",
+        "course_instructorA",
+        "course_staffA",
+    )
+    def test_list_taxonomy_orgA(self, user_attr):
+        """
+        Tests that non staff users from orgA can see only enabled taxonomies from orgA and global taxonomies
+        """
+        expected_taxonomies = ("st1", "t1", "tA1", "tC1")
+        enabled_parameter = True
+        org_parameter = None
+        self._test_list_taxonomy(user_attr, enabled_parameter, org_parameter, expected_taxonomies)
```

Comment:
Yes @rpenido , I think this is an improvement, and your test method comments help a lot.

But I don't think the verbosity helps anything here?  I think it would be better to make your `_test_list_taxonomy` arguments shorter and more legible, so you can call it like this:

```suggestion
        expected_taxonomies = ("st1", "t1", "tA1", "tC1")
        self._test_list_taxonomy(user_attr, enabled=True, org=None, expected=expected_taxonomies)
```